lib/metasploit/cache/module/path/association_extension.rb

```diff
@@ -0,0 +1,71 @@
+# Adds {#each_changed} method to {Metasploit::Cache::Module::Path} associations, so that
+# {Metasploit::Cache::Module::Ancestor} from the given association can be populated from the file system.
+module Metasploit::Cache::Module::Path::AssociationExtension
+  # @note The yielded {Metasploit::Cache::Module::Ancestor} may contain unsaved changes.  It is the responsibility
```

Comment:
This whole docblock looks like copy-pasta from Ancestor.each_changed in app/models/metasploit/cache/module/ancestor.rb; the whole thiing could be a `@see` or a `{include:...}`